d3a_api_client/utils.py

```diff
@@ -126,6 +126,36 @@ def get_area_uuid_from_area_name_and_collaboration_id(collab_id, area_name, doma
     return area_uuid
 
 
+def _name_uuid_mapping_wrapper(serialized_scenario, mapping={}):
```

Comment:
Sorry, but this is not a wrapper, right? This is the function that recursively updates the mapping. So, IMO this naming is misleading, I propose: `_get_name_uuid_mapping` Thanks!